torch/csrc/jit/codegen/cuda/maxinfo_propagator.cpp

```diff
@@ -91,6 +98,20 @@ void MaxInfoSpanningTree::compute_spanning_tree() {
     }
     replayed.emplace(next_hop.to);
 
+    for (auto sibling_tv : ir_utils::siblingTvsOf(next_hop.to)) {
+      if (replayed.count(sibling_tv) ||
+          !allowSibling(next_hop.to, sibling_tv)) {
+        continue;
+      }
+      insertNextHop(
+          {.next_hop =
+               {.type = NextHopType::SIBLING,
+                .from = next_hop.to,
+                .to = sibling_tv},
+           .info_from = next_hop_info.info_to,
```

Comment:
Found the syntax a little confusing on first glance. Might be worth mentioning in a comment that you're just copying the info over, since info has nothing to do with replay state, so sibling info is always identical by definition.

Reply:
Good catch! Actually, I think a better solution is to have a `computeInfoSibling` just like `computeInfoPasC` and `computeInfoCasP`, and add a comment at `computeInfoSibling`. I have implemented this `computeInfoSibling`, let me know what you think.